### PR TITLE
[Movement]: Move absolute with Path Planning 

### PR DESCRIPTION
**Pull request by maltewae**
Absolute movement for multiple stages with collision free path planning.

### CODE REVIEW

File: LabExT/Movement/MoverNew.py [hunk 1]
```diff
@@ -358,6 +360,51 @@ def z_lift(self, lift: float) -> None:
...
+                raise MoverError(
+                    f"No {orientation} stage configured, but target coordinate for {orientation} passed.")
+
+            path_planning.set_stage_target(orientation, target)
+
+        # Move stages on safe trajectory
+        for calibration_waypoints in path_planning.trajectory():
+            for calibration, waypoint in calibration_waypoints.items():
+                with calibration.perform_in_chip_coordinates():
+                    calibration.move_absolute(waypoint, wait_for_stopping)
```
Comment (marcoep) [inline LabExT/Movement/MoverNew.py:406]: Suggestion: all stages/calibrations should move at the same time to their next waypoint (i.e. call `calibration.move_absolute` with `wait_for_stopping=False`. But then wait for all movements completed (incl. SW limited timeout!) before assigning all calibrations a new waypoint.

**Review by marcoep — CHANGES_REQUESTED** [review]

File: LabExT/Movement/MoverNew.py (hunk 1 above)
Comment (maltewae) [reply LabExT/Movement/MoverNew.py:408]: Added!

File: LabExT/Movement/MoverNew.py [hunk 2]
```diff
@@ -358,6 +361,66 @@ def z_lift(self, lift: float) -> None:
...
+
+            # Wait for all stages to stop if stages move simultaneously.
+            if not wait_for_stopping:
+                busy_spinning_start = time()
+                while True:
+                    sleep(0.05)
+
+                    if time() - busy_spinning_start >= wait_timeout:
+                        raise RuntimeError(
+                            f"Stages did not stopped after {wait_timeout}")
```
Comment (marcoep) [inline LabExT/Movement/MoverNew.py:418]: typo

File: LabExT/Movement/MoverNew.py [hunk 3]
```diff
@@ -358,6 +361,66 @@ def z_lift(self, lift: float) -> None:
...
+
+                    if all(
+                            c.stage.is_stopped for c in calibration_waypoints.keys()):
```
Comment (marcoep) [inline LabExT/Movement/MoverNew.py:421]: weird newline

**Review by marcoep — CHANGES_REQUESTED** [review]

File: LabExT/Movement/MoverNew.py (hunk 2 above)
Comment (maltewae) [reply LabExT/Movement/MoverNew.py]: Fixed

File: LabExT/Movement/MoverNew.py (hunk 3 above)
Comment (maltewae) [reply LabExT/Movement/MoverNew.py]: Fixed (kinda)

**Review by marcoep — APPROVED** [review]

**Pull request merged**